app/src/main/java/com/rentminder/MainActivity.kt

```diff
@@ -118,10 +105,57 @@ fun TopToolBar() {
     }
 }
 
-//Bill input boxes, buttons, and text fields
+/**
+ * 2. Column (Calendar Display Name)
+ * Displays current calendar in a month-only format
+ */
+@Composable
+fun CalendarNameColumn()
+{
+    //Getting current month for Main Menu
+    val calendar: Calendar = Calendar.getInstance()
+    val monthDate = SimpleDateFormat("MMMM")
+    val monthName: String = monthDate.format(calendar.time)
+
+    Row(modifier = Modifier
+        .padding(
+            top = 5.dp,
+            bottom = 10.dp
+        )
+        .fillMaxWidth(),
+        horizontalArrangement = Arrangement.Center
+    ) {
+        Text(
+            text = monthName,
+            fontSize = 28.sp,
+            fontWeight = FontWeight.Bold
+        )
+    }
+}
```

Comment:
As described above, this was previously displayed in the main menu function. By implementing this function, developers can view functionality from top to bottom. In this example, I placed CalendarNameColumn function in between the TopNavBarColumn function and the BillColumn to reinforce the layout in the developer's mind.